src/status_im/protocol/web3/inbox.cljs

```diff
@@ -1,67 +1,79 @@
 (ns status-im.protocol.web3.inbox
-  (:require [status-im.constants :as constants]
-            [status-im.protocol.web3.utils :as utils]
-            [status-im.native-module.core :as status]
-            [status-im.protocol.web3.keys :as keys]
-            [taoensso.timbre :as log]))
+  (:require [status-im.native-module.core :as status]
+            [taoensso.timbre :as log]
+            [re-frame.core :as re-frame]
+            [clojure.string :as string]
+            [status-im.protocol.web3.keys :as keys]))
 
-
-;; TODO(oskarth): Determine if this is the correct topic or not
-;; If it is, use constant in other namespace
-(def default-topic "0xaabb11ee")
-
-(def inbox-password "status-offline-inbox")
+(def peers (atom #{}))
+(def trusted-peers (atom #{}))
```

Comment:
What's the reason for putting these in atoms as opposed to in app db? Is it just to keep PR small / change local?

Reply:
Actually, it might be a good idea to persist them to the app db. Otherwise atom state would be reset. But this also could be done in conjunction with https://github.com/status-im/status-react/issues/2751 where user would be allowed to add wnodes (which we will store in app db).

Reply:
Sounds good